### PR TITLE
python310Packages.pytorch-metric-learning: 1.5.0 -> 1.5.1

### PR DESCRIPTION
**Pull request by onny**
###### Description of changes

unbreak, update and enable more tests

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pytorch-metric-learning/default.nix [hunk 1]
```diff
@@ -8,19 +8,20 @@
 , pytorch
 , torchvision
 , tqdm
+, faiss
```
Comment (mweinelt) [inline pkgs/development/python-modules/pytorch-metric-learning/default.nix:11]: I don't think we have `faiss` in nixpkgs.

**Review by mweinelt — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/pytorch-metric-learning/default.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/development/python-modules/pytorch-metric-learning/default.nix:11]: Oh, we have. Since two hours ago.

**Comment by mweinelt** [comment]
> urllib.error.URLError: <urlopen error [Errno -3] Temporary failure in name resolution>

seen on multiple test setups

**Comment by onny** [comment]
@mweinelt Thank you for testing, should be fixed now :)

**Comment by bcdarwin** [comment]
Results of `nixpkgs-review` on x86_64 NixOS:

```
2 packages built:
python310Packages.pytorch-metric-learning python39Packages.pytorch-metric-learning
```

**Review by bcdarwin — APPROVED** [review]
Adding `pythonImportsCheck` would be nice (does this package really no longer depend on faiss at runtime?).

Otherwise LGTM.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 183479 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary>  <ul>    <li>python310Packages.pytorch-metric-learning</li>    <li>python39Packages.pytorch-metric-learning</li>  </ul></details>

**Pull request merged**